### PR TITLE
Create nl.lang

### PR DESCRIPTION
**Pull request by StijnvandeWater**
Added NL lang.
Tested everything incl. error messages.

:)

### CODE REVIEW

**Comment by kolakcc** [comment]
Spotted quite a few mistakes.

File: res/romfs/lang/nl.lang [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+Geen title keys gevonden.\nControleer of de keys in\n%s\n\nKopieer de keys handmatig of controleer\nde instellingen om een\nURL in te voeren voor\nhet downloaden van de keys
+
+No cache entries found\nfor your title keys.\n\nTry refreshing cache in settings.\nIf that doesn't work, then your\ntitles simply won't work with\nfreeShop currently.
+Geen cache gevonden voor\ntitle keys.\n\nProbeer de cache te herladen.\nAls dat niet werkt, dan werken\ndeze title keys niet met\nfreeShop.
+
+Queued install: %s
+Wachtrij: %s
+
+Already installed: %s
+Geïnstalleerd: %s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Perhaps 'Reeds geïnstalleerd'?

File: res/romfs/lang/nl.lang [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+# src/States/DialogState.cpp
+############################
+ Ok
+ Ok
+
+ Cancel
+ Annuleer
+
+Are you sure you want\nto continue?
+Weet je zeker dat je door wilt gaan??
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Should be a single question mark.

File: res/romfs/lang/nl.lang [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+
+# src/States/QrScannerState.cpp
+###############################
+ Retry
+ Opnieuw proberen
+
+# src/AppInfo.cpp
+#################
+No Screenshots
+Geen schermafbeelding
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Should be plural, as the original is: 'Geen schermafbeeldingen'

File: res/romfs/lang/nl.lang [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+
+No game selected
+Geen spel geselecteerd
+
+Demo
+Demo
+
+You are deleting the demo for\nthis title:\n\n%s
+Je verwijderd de demo voor\ndeste título:\n\n%s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Should be the correct verb form: je 'verwijdert', also it seems a copy/paste mistake was made.

File: res/romfs/lang/nl.lang [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+
+Deleted demo: %s
+Demo verwijdered: %s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Should be 'verwijderd'

File: res/romfs/lang/nl.lang [hunk 6]
```diff
@@ -0,0 +1,398 @@
...
+
+Queued demo: %s
+Wachtrij: %s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Should include demo as original translation string: suggest 'Demo wachtrij'

File: res/romfs/lang/nl.lang [hunk 7]
```diff
@@ -0,0 +1,398 @@
...
+
+You are deleting this game,\nincluding all save data:\n\n%s
+Je verwijderd dit spel,\ninclusief al je save data:\n\n%s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Je verwijder**t**

File: res/romfs/lang/nl.lang [hunk 8]
```diff
@@ -0,0 +1,398 @@
...
+Game
+
+Updates
+Updates
+
+DLC
+DLC
+
+You are deleting updates for\nthis title:\n\n%s
+Je verwijderd updates voor\ndeze game:\n\n%s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Je verwijder**t**

File: res/romfs/lang/nl.lang [hunk 9]
```diff
@@ -0,0 +1,398 @@
...
+
+Deleted update: %s
+Update verwijderd: %s
+
+Queued update: %s
+Update in wachtrij: %s
+
+You are deleting DLC for\nthis title:\n\n%s
+Je verwijderd de DLC voor\ndeze game:\n\n%s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Je verwijder**t**

File: res/romfs/lang/nl.lang [hunk 10]
```diff
@@ -0,0 +1,398 @@
...
+Naam
+
+Size
+Grootte
+
+Vote Score
+Stem score
+
+Vote Count
+Stem geteld
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Should be something like 'Aantal stemmen'

**Comment by StijnvandeWater** [comment]
Well.....

I have no words for this.
I really rushed through this.

Thanks for fixing my aweful mistakes.

File: res/romfs/lang/nl.lang [hunk 11]
```diff
@@ -77,16 +77,16 @@ Demo
 Demo
 
 You are deleting the demo for\nthis title:\n\n%s
-Je verwijderd de demo voor\ndeste título:\n\n%s
+Je verwijdert de demo voor\ndeste título:\n\n%s
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: You still forgot 'deste título' :)

**Comment by kolakcc** [comment]
See last comment. Otherwise :+1:.

**Comment by StijnvandeWater** [comment]
Thanks! Let's hope I didn't made any more mistakes. ><

**Comment by kolakcc** [comment]
Err, actually, "Stemmen meetellen" is the incorrect translation of count -- as suggested, I would go for 'aantal'.

**Comment by StijnvandeWater** [comment]
FFS :D

File: res/romfs/lang/nl.lang [hunk 12]
```diff
@@ -0,0 +1,398 @@
...
+Title keys automatisch updaten van URL
+
+Auto-update freeShop
+freeShop automatisch updaten
+
+Refresh Cache
+Cache herladen
+
+Play sound when queue finishes
+Geluid afspelen wanneer voltooid
```
Comment (kolakcc) [inline res/romfs/lang/nl.lang]: Geluid afspelen wanneer wachtrij voltooid

**Comment by kolakcc** [comment]
:+1: for real. Though perhaps you'll have to squash and open a new PR.

**Comment by StijnvandeWater** [comment]
I will, Thanks!

**Pull request closed without merging**